test-fixtures/spago-install-test-deps-success.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  dependencies:
+    - console
+    - effect
+    - prelude
+  name: aaa
+  test:
+    dependencies: [ foreign ]
```

Comment:
I'm not sure how to get this to format better, but presumably because the original content was `[]`, that's why we get `[ foreign ]` rather than
```
dependencies:
  - foreign
```

Thoughts on how to fix that?

Reply:
I had no idea so I checked in the node repl:
```node
> yaml = require('yaml');
> let d1 = yaml.parseDocument("- foreign")
> let d2 = yaml.parseDocument("[foreign]")
> d1.contents
YAMLSeq {
  items: [
    Scalar {
      value: 'foreign',
      range: [Array],
      source: 'foreign',
      type: 'PLAIN'
    }
  ],
  range: [ 0, 9, 9 ]
}
> d2.contents
YAMLSeq {
  items: [
    Scalar {
      value: 'foreign',
      range: [Array],
      source: 'foreign',
      type: 'PLAIN'
    }
  ],
  flow: true,
  range: [ 0, 9, 9 ]
}
```

It looks like should set the [flow style](https://www.yaml.info/learn/flowstyle.html) to false whenever we modify collections